### PR TITLE
Implement Framework Config

### PR DESCRIPTION
**Pull request by OR13**

### CODE REVIEW

File: src/EventStore/EventStore.ts [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+import { TransmuteFramework }  from '../TransmuteFramework'
 
-import { web3 } from '../env'
+
+// const { web3 } = TransmuteFramework
+// .init({
+//     env: 'testrpc'
+// })
+
+let web3
```
Comment (eolszewski) [inline src/EventStore/EventStore.ts]: @OR13 unused.

Comment (eolszewski) [inline src/EventStore/EventStore.ts]: Everything above this is unused.

File: src/EventStore/EventStore.spec.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
 'use strict'
 
-const contract = require('truffle-contract')
+import TransmuteFramework from '../TransmuteFramework'
 
+const { web3, EventStoreContract } = TransmuteFramework.init()
```
Comment (eolszewski) [inline src/EventStore/EventStore.spec.ts]: 🌶

File: src/EventStore/EventStoreFactory/EventStoreFactory.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 'use strict'
 
-import { web3 } from '../../env'
+import TransmuteFramework from '../../TransmuteFramework'
+
+const { web3 } = TransmuteFramework
```
Comment (eolszewski) [inline src/EventStore/EventStoreFactory/EventStoreFactory.ts]: @OR13 .init() isn't needed here?

File: src/EventStore/EventTypes/EventTypes.ts [hunk 4]
```diff
@@ -132,7 +130,7 @@ export module EventTypes {
     }
 
     export const toAscii = (value) => {
-        return web3.toAscii(value).replace(/\u0000/g, '')
+        return ethUtils.toAscii(value).replace(/\u0000/g, '')
```
Comment (eolszewski) [inline src/EventStore/EventTypes/EventTypes.ts]: That's nice.

File: src/EventStore/Middleware/Middleware.ts [hunk 5]
```diff
@@ -1,6 +1,11 @@
 
+// import TransmuteFramework from '../../TransmuteFramework'
+
+// const { web3 } = TransmuteFramework
+
+let web3
```
Comment (eolszewski) [inline src/EventStore/Middleware/Middleware.ts]: Use it or lose it.

File: src/EventStore/Mock/Events/TestEvents.ts [hunk 6]
```diff
@@ -1,12 +1,21 @@
-import { web3 } from '../../../env'
+// import TransmuteFramework from '../../../TransmuteFramework'
+
+// const { web3 } = TransmuteFramework
+
+let web3
```
Comment (eolszewski) [inline src/EventStore/Mock/Events/TestEvents.ts]: Remove.

File: src/TransmuteFramework.spec.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        it('should use testrpc as default web3 provider', async () => {
+            let { web3 } = TransmuteFramework.init()
+            assert (web3._requestManager.provider.host === 'http://localhost:8545')
+        })
+
+        it('should support infura ropsten', async () => {
+            const eventStoreArtifacts = require('../build/contracts/EventStore')
+            const eventStoreFactoryArtifacts = require('../build/contracts/EventStoreFactory')
+
+            let { web3 } = TransmuteFramework.init({
```
Comment (eolszewski) [inline src/TransmuteFramework.spec.ts]: 🌶 🐸

File: src/env.ts [hunk 8]
```diff
@@ -1,20 +0,0 @@
...
-  case 'infura': _provider = 'https://ropsten.infura.io'; break
-}
-
-switch (currentEnv) {
-  case 'testrpc': _web3 = new Web3(new Web3.providers.HttpProvider(_provider)); break
-  case 'parity': _web3 = new Web3(new Web3.providers.HttpProvider(_provider)); break
-  case 'metamask': _web3 = window.web3; break
-}
-
-export const web3 = _web3
```
Comment (eolszewski) [inline src/env.ts]: You will be missed, env.ts

**Comment by OR13** [comment]
@eolszewski I implemented ur feedback

**Pull request merged**